data/extension/init/templates/rust/envoy.access_loggers/default/src/logger.rs

```diff
@@ -67,11 +65,9 @@ impl<'a> AccessLogger for SampleAccessLogger<'a> {
         // Update stats
         self.stats.log_entries_total().inc()?;
 
-        let now: DateTime<Local> = self.clock.now()?.into();
-
         log::info!(
-            "logging at {} with config: {:?}",
-            now.format("%+"),
+            "logging at {:?} with config: {:?}",
+            self.clock.now(),
```

Comment:
```suggestion
            self.clock.now()?,
```

This would make the output a little bit less ugly from `Ok(SystemTime(1618809891.154186s))` to `SystemTime(1618809891.154186s)`

Reply:
thanks.. I'll apply to the other places also.